src/index.html

```diff
@@ -7,12 +7,74 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/style.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="Apple iMac Monoblock"
+        class="product-card__image"
+      />
+      <h2 class="product-card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="product-card__description">Product code: 195434</p>
+      <div class="product-card__rate">
+        <div class="product-card__rate product-card__rate--stars">
+          <div
+            class="product-card__rate product-card__rate--stars--items"
+          ></div>
+          <div
+            class="product-card__rate product-card__rate--stars--items"
+          ></div>
+          <div
+            class="product-card__rate product-card__rate--stars--items"
+          ></div>
+
+          <div
+            class="product-card__rate product-card__rate--stars--items"
+          ></div>
+          <div
+            class="product-card__rate product-card__rate--stars--items"
+          ></div>
+        </div>
```

Comment:
You need to use stars block from the previous task
```suggestion
        <div class="stars stars--4">
          <div class="stars__stars"></div>
          <div class="stars__stars"></div>
          <div class="stars__stars"></div>
          <div class="stars__stars"></div>
          <div class="stars__stars"></div>
        </div>
```